docs/developers/cosmwasm-environment.md

```diff
@@ -38,14 +251,20 @@ STAKING_AMOUNT=1000000000uwasm
 wasmd gentx $KEY_NAME $STAKING_AMOUNT --chain-id $CHAIN_ID --keyring-backend test
 ```
 
-With that, we created a local network genesis file.
+With that, we created a local network genesis file. Our output will look
+similar to:
+
+```sh
+Genesis transaction written to "/root/.wasmd/config/gentx/gentx-4dec90c32140dbe758613d1f5f8e78f83243ab9a.json"
+```
 
 Some more useful commands we can setup:
 
 <!-- markdownlint-disable MD013 -->
 ```sh
 export NODE="--chain-id ${CHAIN_ID}"
 export TXFLAG="--chain-id ${CHAIN_ID} --gas-prices 0uwasm --gas auto --gas-adjustment 1.3"
+export NODEIP="--node http://127.0.0.1:26657"
```

Comment:
are we still on port 26675 or are we using 26659 now?

Reply:
i'm fairly certain this is the right port for this node flag, because i just ran through it in last day

Reply:
i just ran through this again with no issues having nodeip set to that port